player/main.c

```diff
@@ -487,6 +488,11 @@ int mpv_main(int argc, char *argv[])
 {
     struct MPContext *mpctx = mp_create();
 
+#if HAVE_MACOS_TOUCHBAR
+    mpv_handle *ctx = mp_new_client(mpctx->clients, "osx");
+    cocoa_set_mpv_handle(ctx);
+#endif
```

Comment:
This is an example for code which could be always enabled, even if it does nothing if no touchbar support is compiled.